README.md

```diff
@@ -4,6 +4,7 @@
 [![Build Status](https://travis-ci.org/jupyter-widgets/ipywidgets.svg?branch=master)](https://travis-ci.org/jupyter-widgets/ipywidgets)
 [![Documentation Status](http://readthedocs.org/projects/ipywidgets/badge/?version=latest)](https://ipywidgets.readthedocs.io/en/latest/?badge=latest)
 [![Join the chat at https://gitter.im/ipython/ipywidgets](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/jupyter-widgets/Lobby)
+[![Binder](https://mybinder.org/badge.svg)](https://mybinder.org/v2/gh/ashwani-pandey/ipywidgets/master?filepath=docs%2Fsource%2Fexamples)
```

Comment:
Thanks! Would you switch this link to point to the official repository, `.../gh/jupyter-widgets/ipywidgets/....`?